extensions/sql-database-projects/src/dialogs/insertSqlBindingQuickpick.ts

```diff
@@ -0,0 +1,76 @@
+import * as vscode from 'vscode';
+import * as constants from '../common/constants';
+
+export async function launchInsertSqlBindingQuickpick(uri: vscode.Uri): Promise<void> {
+	if (!uri) {
+		// this command only shows in the command palette when the active editor is a .cs file, so we can safely assume that's the scenario
+		// when this is called without a uri
+		uri = vscode.window.activeTextEditor!.document.uri;
+	}
+
+	// 1. select input or output binding
+	const intputOutputItems: vscode.QuickPickItem[] = [{ label: constants.input }, { label: constants.output }];
```

Comment:
inputOutputItems

Reply:
Also you can just pass in an array of strings - no reason to make them quickpick items unless you want to tie additional information to them as an object.